src/common/regions/regions_data.ts

```diff
@@ -69,12 +70,14 @@ export const getStateFips = (region: Region): string | null => {
 function buildStates(): State[] {
   return chain(state_dataset)
     .map(stateInfo => {
+      const ccviData = fipsToCcviData[stateInfo.state_fips_code];
       return new State(
         stateInfo.state,
         stateInfo.state_url_name,
         stateInfo.state_fips_code,
         stateInfo.population,
         stateInfo.state_code,
+        ccviData || {},
```

Comment:
It might be better to default to `null` so we can tell if we don't have CCVI data for a location before we try to access the themes.